Algo3 fails on KeyError for 'emoticons'

When running the algos for https://www.twitch.tv/videos/1015750167 , algo3 errors out here: 

```
[ERROR] KeyError: 'emoticons'Traceback (most recent call last):  File "/var/task/handler.py", line 33, in handler    algo3 = algo3_0.run(all_messages, min_=.5,  limit=10)  File "/var/task/pillaralgos/algo3_0.py", line 141, in run    results, first_stamp = thalamus(big_df, min_=min_, min_words=min_words, goal="num_top_user_appears")  File "/var/task/pillaralgos/algo3_0.py", line 18, in thalamus    id_words = id_words_counter(big_df)  File "/var/task/pillaralgos/algo3_0.py", line 89, in id_words_counter    emoji = temp_df["emoticons"].apply(lambda x: 0 if type(x) == float else len(x))  File "/var/task/pandas/core/frame.py", line 3024, in __getitem__    indexer = self.columns.get_loc(key)  File "/var/task/pandas/core/indexes/base.py", line 3082, in get_loc    raise KeyError(key) from err
--

```

--- a/prod/pillaralgos/algo3_0.py
+++ b/prod/pillaralgos/algo3_0.py
@@ -86,8 +86,11 @@ def id_words_counter(big_df):
     for _id in big_df["_id"].unique():
         temp_df = big_df[big_df["_id"] == _id]
         words = temp_df["body"].str.split(" ")
-        emoji = temp_df["emoticons"].apply(lambda x: 0 if type(x) == float else len(x))
-        num_emoji = emoji.sum()
+        try: 
+            emoji = temp_df["emoticons"].apply(lambda x: 0 if type(x) == float else len(x))
+            num_emoji = emoji.sum()
+        except:
+            num_emoji = 0.0
         num_words = words.apply(lambda x: len(x))
 
         sum_words = num_words.sum()

